commit ed8efcf9263d3ed12b201de5b4a4eb10df77b81c
Author: maxice8 <thinkabit.ukim@gmail.com>
Date:   Sat Feb 23 05:55:00 2019 -0300

    meson.build: use program from build machine not host or target.
    
    We can't always execute binaries from the host or target machine,
    as is the case in cross compilation.
    
    closes #89

diff --git meson.build meson.build
index 9fef9c3..6f0b596 100644
--- meson.build
+++ meson.build
@@ -411,7 +411,7 @@ endif
 if get_option('enable-wayland')
     wayland_client_dep = dependency('wayland-client', version: '>=1.2.0', required: false)
     wayland_protocols_dep = dependency('wayland-protocols', version: '>=1.7', required: false)
-    wayland_scanner_dep = dependency('wayland-scanner', required: false)
+    wayland_scanner_dep = dependency('wayland-scanner', required: false, native: true)
     if not wayland_client_dep.found() or not wayland_protocols_dep.found() or not wayland_scanner_dep.found()
         error('''The Wayland demo programs require wayland-client >= 1.2.0, wayland-protocols >= 1.7 which were not found.
 You can disable the Wayland demo programs with -Denable-wayland=false.''')

